financial-connections/src/main/java/com/stripe/android/financialconnections/features/consent/ui/ConsentLogoHeader.kt

```diff
@@ -136,33 +143,14 @@ private fun debugPreviewBitmaps(
 }
 
 @Composable
-private fun loadBitmaps(
-    logos: List<String>,
-    bitmapLoadSize: Int,
-    isVisible: MutableState<Boolean>
-): SnapshotStateList<ImageBitmap?> {
-    val stripeImageLoader = LocalImageLoader.current
-    val loadedImages = remember { mutableStateListOf<ImageBitmap?>() }
-    // Load images
-    logos.forEachIndexed { index, logo ->
-        LaunchedEffect(logo) {
-            stripeImageLoader.load(
-                logo,
-                bitmapLoadSize,
-                bitmapLoadSize
-            ).onSuccess { result ->
-                while (loadedImages.size <= index) {
-                    loadedImages.add(null)
-                }
-                loadedImages[index] = result?.asImageBitmap()
-                // Set visibility to true when all images are loaded
-                if (loadedImages.size == logos.size && loadedImages.all { it != null }) {
-                    isVisible.value = true
-                }
-            }
-        }
-    }
-    return loadedImages
+private fun rememberPlaceholderBitmap(bitmapLoadSize: Int, placeholderColor: Color): ImageBitmap = remember {
```

Comment:
Nit: Unlikely to change, but we should use `remember(bitmapLoadSize, placeholderColor)`.